pkg/controller/middleware.go

```diff
@@ -100,10 +100,8 @@ func (c *Controller) authenticateUser(ctx iris.Context) (err error) {
 		}()
 	}
 
-	if token = ctx.GetCookie(TokenName, iris.CookieSecure); token != "" {
+	if token = ctx.GetCookie(TokenName, iris.CookieSecure, iris.CookieHTTPOnly(true)); token != "" {
```

Comment:
This should also use the configuration, if the cookie not secure (someone is just using http because its locally hosted) then we can't only look for secure cookies.